### PR TITLE
feature/FOUR-13370: The Avatar is not working correctly as Participant and Designer package

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps

The Avatar is not the same in Home Option and Designer Option, since is the same user that was login.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13370

## PR Related in the core
https://github.com/ProcessMaker/processmaker/pull/6190

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy
ci:processmaker:feature/FOUR-13370

### CODE REVIEW

File: src/components/renderer/form-avatar.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      return this.user.firstname && this.user.lastname
+        ? this.user.firstname.match(/./u)[0] + this.user.lastname.match(/./u)[0]
+        : "";
+    },
+    variant() {
+      if (this.user.avatar) {
+        return 'secondary';
+      } else {
+        return 'info';
+      }
```
Comment (fagubla) [inline src/components/renderer/form-avatar.vue:58]: `return this.user.avatar ? "secondary" : "info";`

File: src/components/renderer/form-avatar.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    },
+    styleAvatar() {
+      return "width: " +
+        this.width +
+        "px; height: " +
+        this.height +
+        "px; font-size:" +
+        this.height / 2.5 +
+        "px; padding:0;";
```
Comment (fagubla) [inline src/components/renderer/form-avatar.vue:67]:
```javascript
return (
        `width: ${this.width}px; ` +
        `height: ${this.height}px; ` +
        `font-size:${this.height / 2.5}px; ` +
        "padding:0;"
      );
```

**Review by fagubla — APPROVED** [review]

File: src/components/renderer/form-avatar.vue [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      class="avatar-button rounded-circle overflow-hidden p-0 m-0 d-inline-flex border-0"
+      disabled
+    >
+      <img v-if="user.avatar"
+        :src="user.avatar"
+        :width="width"
+        :height="height"
+        :class="image"
+        :alt="user.fullname"
+      >
```
Comment (fagubla) [inline src/components/renderer/form-avatar.vue:15]: must be />

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by fagubla** [comment]
@pmPaulis It'is working as the expected
[recording (1).webm](https://github.com/ProcessMaker/screen-builder/assets/5005237/4f652afc-2742-43e0-a5f5-bdb292b82705)

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-424bf95af9.engk8s.processmaker.net

**Pull request merged**